ingress/single-cluster/ingress-cloudarmor/run-test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+recipe_dir=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P ) || exit 1
+source "$(find ${recipe_dir}/../../../test/ -name helper.sh)" "$@"
+
+resource_yaml="$(find -name cloudarmor-ingress.yaml)" || exit 1
+namespace="ingress-cloudarmor"
+kubectl create namespace "${namespace}" || exit 1
```

Comment:
you can set global settings for the script to avoid having to do this on every command:

https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425